### PR TITLE
Try mono 6

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
See https://stackoverflow.com/questions/19456518/invalid-command-code-despite-escaping-periods-using-sed about the OSX error

**Comment by Tobias-Fischer** [comment]
Hi @wolfv,
Did you have any chance to look further into this? Is it just the broken OSX build which stops this PR from being ready to be merged? I can fix that if you want to.

**Comment by Tobias-Fischer** [comment]
Ping @wolfv

**Review by chrisburr — APPROVED** [review]

File: recipe/recipe.yaml
Comment (isuruf) [inline recipe/recipe.yaml]: What is this file doing here?

File: recipe/build.sh
Comment (isuruf) [inline recipe/build.sh]: Use `target_platform`

File: recipe/build.sh
Comment (isuruf) [inline recipe/build.sh]: Why do you need these?

File: recipe/build.sh
Comment (isuruf) [inline recipe/build.sh]: If this is vendoring glib, you need to package their license/copyright notices

File: recipe/recipe.yaml
Comment (wolfv) [reply recipe/recipe.yaml]: nothing, has to be removed :) wanted to get a review before trigggering another build.

File: recipe/build.sh
Comment (wolfv) [reply recipe/build.sh]: I copied this from the RPM I found. DYLD path is definitely not needed, I can check if it works wihtut LD_...

**Comment by Tobias-Fischer** [comment]
Looks like this is ready to be merged @wolfv @isuruf? :)

File: recipe/meta.yaml
Comment (isuruf) [inline recipe/meta.yaml]: Why do you need `gettext` in host?

Comment (Tobias-Fischer) [reply recipe/meta.yaml]: As far as I can see, `gettext` is in host already (it's not added in this PR). `gettext` is used quite a bit throughout mono (see https://github.com/mono/mono/search?q=gettext).

It would be great to see this PR merged :)

**Comment by Tobias-Fischer** [comment]
Another ping @isuruf @wolfv :)

File: recipe/meta.yaml
Comment (isuruf) [reply recipe/meta.yaml]: Not really. That's a different thing.

**Comment by isuruf** [comment]
You should ping @bgruening who is the maintainer of this feedstock.

**Comment by wolfv** [comment]
pinging @bgruening :)

**Comment by Tobias-Fischer** [comment]
Another ping @bgruening

**Comment by isuruf** [comment]
I'll merge if you fix the `gettext` dependency.

**Comment by Tobias-Fischer** [comment]
Looks like it's ready @isuruf 👍

File: recipe/build.sh
Comment (isuruf) [inline recipe/build.sh]:
```suggestion
```

**Comment by Tobias-Fischer** [comment]
It seems like removing `disable-silent-rules` leads to issues on ARM. Or is it unrelated?!?

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please restart ci

**Comment by Tobias-Fischer** [comment]
Seems like it is working fine now, so I think this is good to merge @wolfv @isuruf @bgruening

**Comment by Tobias-Fischer** [comment]
Ping @wolfv @bgruening

**Comment by Tobias-Fischer** [comment]
Another ping @conda-forge/mono @wolfv :)

**Comment by isuruf** [comment]
We can fix travis-ci later

**Pull request merged**
